docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM composer as composer
```

Comment:
I'd name the stage something different so it doesn't collide with the official image